### PR TITLE
manifest: Update sdk-zephyr

### PR DESCRIPTION
**Pull request by hermabe**
Includes various Bluetooth Host bug fixes and documentation
improvements.

Signed-off-by: Herman Berget <herman.berget@nordicsemi.no>

### CODE REVIEW

**Comment by rlubos** [comment]
@hermabe Could you rebase this PR? In the previous run BLE tests failed.

**Comment by hermabe** [comment]
> @hermabe Could you rebase this PR? In the previous run BLE tests failed.

https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsub%2Ftest-fw-nrfconnect-ble/detail/master/19117/pipeline

They did not. There were some device issues and part of the job was automatically retried on a different test node and passed. However, the job was not marked as passed, even though all parts passed. See `ble_controller_5340app` in the above linked pipeline,

The TFM job failed. but that should be unrelated to the Bluetooth Host changes and fixed by the TFM team now after https://github.com/nrfconnect/sdk-nrf/pull/8621 was merged.

**Comment by rlubos** [comment]
> They did not.

Well, all I saw was:
image

But I trust your judgment here.

**Comment by hermabe** [comment]
> > They did not.
> 
> Well, all I saw was: image
> 
> But I trust your judgment here.

Yes, there is an issue where the retry of a step on a different node did not make the entire job green. If you click through to the BLE job you can see all the steps are green.

File: west.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 6e460c0847d437abb3b519bca048fafccbdd9411
+      revision: pull/901/head
```
Comment (rlubos) [inline west.yml:53]: Please update SHA: `3db330717c1ca3ec4fb133400bf3eb7156aa6cbd`

**Review by rlubos — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Comment by rlubos** [comment]
CI passed on rerun.

**Pull request merged**